hca/define_api.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+# import os, sys, json, time, base64
+import json, sys, os
+from io import open
+# import pprint
+import requests
+from .parser import get_parser
+
+
+class Api:
+    """Class for interacting with the REST Api."""
```

Comment:
API should probably be all caps throughout. Also, "class API" is not really descriptive. I think what this class does is build a parser, right? How about calling it ParserBuilder? And the docstring could then say something like "Given a Swagger API definition, builds an argparse parser for a CLI".